### PR TITLE
Added E2E tests for #12585.

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [x] E2E Tests for user-facing bug

## Description

Adds end to end tests for #12585

## Related Tickets & Documents

#12585

## QA Instructions, Screenshots, Recordings

On CI, E2E tests should pass.

To test locally, follow the instructions in the [E2E tests documentation](https://docs.forem.com/tests/e2e-tests/).

### UI accessibility concerns?

N/A

## Added tests?

- [x] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## Added to documentation?

- [ ] [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)
- [ ] README
- [x] No documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

A test screen

### CODE REVIEW

**Review by katiedavis — APPROVED** [review]

File: cypress/integration/publishingFlows/previewArticle.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        });
+      });
+    });
+
+    it('should preview blank content of an article', () => {
+      cy.findByTestId('article-form').as('articleForm');
+
+      cy.get('@articleForm')
+        .findByText(/^Preview$/i)
+        .click();
```
Comment (fdoxyz) [inline cypress/integration/publishingFlows/previewArticle.js:55]: **Question:** Is this test missing a `should` matcher to ensure no errors are visible? Something like the other test below but along the lines of

```js
// Notice how I don't know the exact syntax of the "should_not" :)
cy.findByTestId('error-message').should_not('be.visible');
```

Without knowing much about Cypress it feels like if an error does bubble up again we won't be able to catch it but I may be totally wrong

Comment (nickytonline) [reply cypress/integration/publishingFlows/previewArticle.js:55]: Good catch. I'll add a check to ensure the error is not visible in the successful path.

Comment (nickytonline) [reply cypress/integration/publishingFlows/previewArticle.js:56]: Done.

**Review by fdoxyz — APPROVED** [review]
Yay for more test coverage! Thanks @nickytonline

File: spec/support/seeds/seeds_e2e.rb [hunk 2]
```diff
@@ -12,23 +12,6 @@
 SiteConfig.public = true
 SiteConfig.waiting_on_first_user = false
 
-seeder.create_if_none(Organization) do
```
Comment (nickytonline) [inline spec/support/seeds/seeds_e2e.rb:15]: We do not need organizations in the seed data for E2E tests. At least not yet.

File: spec/support/seeds/seeds_e2e.rb [hunk 3]
```diff
@@ -84,3 +67,47 @@
 end
 
 ##############################################################################
+
+seeder.create_if_doesnt_exist(User, "email", "article-editor-v1-user@forem.com") do
```
Comment (nickytonline) [inline spec/support/seeds/seeds_e2e.rb:71]: Test users for the v1 and v2 editor.

**Pull request merged**